summary-visualiser/src/index.js

```diff
@@ -0,0 +1,97 @@
+import fs from "node:fs";
+import path from "node:path";
+import handlebars from "handlebars";
+import scenarioTransforms from "./scenarioTransforms.js";
+import { fileURLToPath } from "node:url";
+import { formatNumber } from "./util.js";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const templatesFolderPath = path.join(__dirname, "templates");
+const partialsFolderPath = path.join(templatesFolderPath, "partials");
+
+// Load all the partials and make them available to the template.
+const partials = fs.readdirSync(partialsFolderPath);
+for (const p of partials) {
+    const partialName = p.replace(".hbs", "");
+    const partialContent = fs.readFileSync(path.join(partialsFolderPath, p), { encoding: "utf8" });
+    handlebars.registerPartial(partialName, partialContent);
+}
+
+// A few helpers
```

Comment:
I know handlebars doesn't really allow logic in templates by-design, and the helpers are how you work around that. Would it make more sense to use a template system that allows logic?

Reply:
Yes! I'm switching to `gomplate` which also seems logicless but has an absolute heap of helper functions. It does mean you need to do Polish notation for comparisons and arithmetic, but it means less code for us to maintain.